.github/workflows/master.yml

```diff
@@ -33,7 +33,22 @@ jobs:
   r:
     runs-on: ubuntu-latest
     steps:
+    - name: Uninstall default-jdk and adoptopenjdk-11-hotspot if present
+      run: |
+       # deleting other version(s) of JDK because they are not needed and they might interfere with JNI linker configuration in the 'setup-r' step
+       sudo apt-get -y remove --purge default-jdk adoptopenjdk-11-hotspot || :
```

Comment:
```suggestion
        # deleting other version(s) of JDK because they are not needed and they might interfere with JNI linker configuration in the 'setup-r' step
        sudo apt-get -y remove --purge default-jdk adoptopenjdk-11-hotspot || :
```

nit: indent